app/battle/batlle_of_knights.py

```diff
@@ -1,9 +1,9 @@
-from app.knight.knight import Knight
+from app.knight.knight_model import Knight

```

Comment:
Before was better 
knight > knight_model
Okay. If you need choose or `knight` or `model`)))